docs/pages/enroll-resources/database-access/enroll-self-hosted-databases/oracle-exadata.mdx

```diff
@@ -0,0 +1,428 @@
+---
+title: Database Access with Oracle Exadata
+description: How to configure Teleport database access with Oracle Exadata.
+---
+
+(!docs/pages/includes/database-access/self-hosted-introduction.mdx dbType="Oracle Exadata"!)
+
+## How it works
+
+(!docs/pages/includes/database-access/how-it-works/mtls.mdx db="Oracle"!)
+
+<Tabs>
+<TabItem scope={["enterprise"]} label="Teleport Enterprise">
+![Teleport Database Access Self-hosted Oracle](../../../../img/database-access/guides/oracle_selfhosted.png)
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+![Teleport Database Access Oracle Cloud](../../../../img/database-access/guides/oracle_selfhosted_cloud.png)
+</TabItem>
+
+</Tabs>
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- Oracle Exadata server instance 19c or later.
+- The `sqlcl` [Oracle client](https://www.oracle.com/pl/database/sqldeveloper/technologies/sqlcl/) installed and added to your system's `PATH` environment variable or any GUI client that supports JDBC
+  Oracle thin client.
+
+
+## Step 1/6. Create a Teleport token and user
```

Comment:
Can this step be moved after "Configure Oracle Exadata"? Otherwise the flow of the guide is switching between Teleport and database configuration. I don't think we're creating anything in this step that will be needed for the database setup.

Reply:
I didn't want to deviate from the pattern set by other guides, but I agree with you. Reorganising the steps will lead to less context switching in the guide.

Reply:
Reordered.